pkg/operator/ceph/controller/spec.go

```diff
@@ -46,6 +47,7 @@ const (
 	logCollector                          = "log-collector"
 	DaemonIDLabel                         = "ceph_daemon_id"
 	daemonTypeLabel                       = "ceph_daemon_type"
+	DefaultServiceAccount                 = "rook-ceph-default"
```

Comment:
```suggestion
	DefaultServiceAccount                 = "rook-ceph-default"  // DefaultServiceAccount is a service account used for components that do not specify a dedicated service-account.
```

Reply:
It's preferred to add the comment on the line above instead of at the end

Reply:
@parth-gr ^^